vite.config.ts

```diff
@@ -42,7 +42,8 @@ export default defineConfig(({ command, mode }) => {
             },
             rollupOptions: {
               plugins: [rollupNodePolyFill()]
-            }
+            },
+            sourcemap: true
```

Comment:
Another way could be passing `-- --sourcemap ${{ github.event_name == 'release' && github.event.action == 'created' }}` in the GitHub Action step that runs `npm run build`.

Any thoughts @LautaroPetaccio?

We need those sourcemaps to upload them to Sentry. It is not mandatory for the rest of our builds, tho.